htdocs/takepos/reduction.php

```diff
@@ -197,14 +202,33 @@ function ValidateReduction()
 			});
 		} else {
 			console.error('Error bad reduction type :', reductionType);
+			return false
+		}
+
+		return true;
+	}
+
+	$(document).ready(function() {
```

Comment:
Do you still need this now you added the "autofocus" property on the input field reduction_total ?

Reply:
ok i remove the focus call; ready to merge.